use/events.js

```diff
@@ -196,6 +200,8 @@ export const useEvents = () => {
       label: t('event.startDate'),
       simple: true,
       onChange: updateEndDate,
+      validation: (val) => !!val,
+      validationErrorMessage: 'please enter a valid event start date',
```

Comment:
Issue requirement has a different name of that field:

```suggestion
      validationError: 'please enter a valid event start date',
```